packages/blocks/src/lib/navigation-map/components/nav/obstacles.svelte

```diff
@@ -183,9 +183,7 @@ $: debugMode = $environment === 'debug';
   <li
     class="group sticky bottom-0 z-10 bg-white pt-4"
     on:mouseenter={() => {
-      if (selectedObstacle) {
-        $hovered = selectedObstacle.name;
-      }
+      $hovered = selectedObstacle.name;
```

Comment:
unnecessary-conditional was firing -- this is wrapped in a `{#if selectedObstacle}`